packages/cozy-sharing/src/SharingProvider.jsx

```diff
@@ -217,12 +217,15 @@ export class SharingProvider extends Component {
         readOnlyRecipients
       })
 
+    const previewPath =
+      document._id === 'com.bitwarden.organizations' ? undefined : '/preview'
```

Comment:
I would like to extract this `com.bitwarden.organizations` to a const. Ideally in a file accessible from here but also from this file : https://github.com/cozy/cozy-client/pull/990/files#diff-eedaf5c6b0118a4e2a44524e49bf386befb317815ee159d97a2242a578b96724

Does someone know if this place exists? Maybe `cozy-doctypes` ?

Reply:
The future of cozy-doctypes doesn't exist. We want to shutdown this lib. The content should be migrated to cozy-client. We have done some work (in cozyclient/helpers) but still remaining a lot of stuff. 

I think cozy-client is the right place to store this kind of information

Reply:
@Crash--  `com.bitwarden.organizations` is also declared in `cozy-stack-client` here : https://github.com/cozy/cozy-client/pull/990/files#diff-eedaf5c6b0118a4e2a44524e49bf386befb317815ee159d97a2242a578b96724R5-R7

Currently:
- `cozy-sharing` imports `cozy-client` but not `cozy-stack-client`
- `cozy-stack-client` doest not import `cozy-client`
- `cozy-client` imports `cozy-stack-client`

Is it ok if I import `cozy-stack-client` package in `cozy-sharing`?
Or do you still prefer to add those doctypes in `cozy-client`? -> I would need to import `cozy-client` in `cozy-stack-client` so.

Reply:
That's a lot of thinking for not much. And since we'll move this string to the caller, no need to think about it anymore :D

Reply:
https://www.youtube.com/watch?v=18SNR9c09is